package.json

```
@@ -5,6 +5,8 @@
     "prepare": "wsrun -m prepare",
     "lint": "eslint",
     "dev:ui": "yarn --cwd ./packages/ui dev",
+    "dev:api": "yarn --cwd ./packages/ui dev:api",
+    "dev": "concurrently --kill-others-on-fail --names \"client,server\" --prefix-colors \"magenta.bold,green.bold\" \"yarn dev:ui\" \"yarn dev:api\"",
```

Perhaps name as ui,api to make it clearer
```suggestion
    "dev": "concurrently --kill-others-on-fail --names \"ui,api\" --prefix-colors \"magenta.bold,green.bold\" \"yarn dev:ui\" \"yarn dev:api\"",
```